### PR TITLE
Home button web fixes

### PR DESCRIPTION
**Pull request by gwsii**
Revisited application styling and addressed mobile layout.
Fix for empty image on Product detail page.
Auto-login fixes.
Updated styling for mobile app.
Prefer user's ID over e-mail address.

### CODE REVIEW

**Comment by gwsii** [comment]
@laurwang Please be sure that I did not delete any of the schemas you needed during my merge. Thanks!

File: retail-stream/retailStreamApi.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+const aws = require('aws-sdk') // eslint-disable-line import/no-unresolved, import/no-extraneous-dependencies
+
+// TODO Make a dynamoDB in a service that holds all of the schema and a schema-getter and validator, instead of listing them out here
+const AJV = require('ajv')
+
+const ajv = new AJV()
+const makeSchemaId = schema => `${schema.self.vendor}/${schema.self.name}/${schema.self.version}`
+
+const productPurchaseSchema = require('./schemas/product-purchase-schema.json')
+const productCreateSchema = require('./schemas/product-create-schema.json')
```
Comment (erikerikson) [inline retail-stream/retailStreamApi.js]: product-create-schema appears deleted above?

**Comment by erikerikson** [comment]
this contains old stuff greg.

File: retail-stream/retailStreamApi.js [hunk 2]
```diff
@@ -0,0 +1,180 @@
+'use strict'
```
Comment (erikerikson) [inline retail-stream/retailStreamApi.js]: this file is [here](https://github.com/Nordstrom/hello-retail/tree/develop/event-writer) in develop

Comment (erikerikson) [inline retail-stream/retailStreamApi.js]: it appears you need to fetch and merge or something like that...

**Pull request closed without merging**